rules/src/main/scala/fix/UseNamedParameters.scala

```diff
@@ -102,9 +102,11 @@ final class UseNamedParameters(config: UseNamedParametersConfig)
                 case Some(symInfo)
                     // In the case of repeated parameters, if the parameter name is given only at the beginning, it is broken.
                     if !symInfo.signature.toString().startsWith("* Tuple") =>
-                  Term.Name(symInfo.displayName).toString match {
-                    case pattern() if config.skipSingleAlphabet => Patch.empty
-                    case paramName => Patch.addLeft(t, s"$paramName = ")
+                  val paramName = Term.Name(symInfo.displayName).toString
+                  if (config.skipSingleAlphabet && paramName.matches(pattern.pattern.pattern())) {
```

Comment:
```suggestion
                  if (config.skipSingleAlphabet && pattern.matches(paramName)) {
```
😄 
Also recommend giving a more descriptive name to `pattern`. Perhaps "singleAlphabetPattern"

Reply:
Pattern#matches is from Scala 2.13.
It probably won't compile with this project.

Reply:
Good point  I didn't know that :) 
How about defining `private val singleAlphabetPattern = "^[A-Za-z]\\d*$".r` and use singleAlphabetPattern.findFirstIn?

Because right now the code is recompiling the regex every time it's called. (`pattern.pattern.pattern` returns a regular expression as String, which has to be recompiled by String#matches)

Reply:
I completely overlooked that `Pattern#pattern` returns a string :bow: I would like to make a correction!